### DISCUSSION TITLE
Buchung mehreren Sollstellungen zuweisen -> automatische Splittbuchung

### DISCUSSION THREAD

**Discussion by fkuersch** [openjverein/jverein#461] [Ideas]
Zahlt ein Mitglied mehrere Monatsbeiträge nach, wäre es sehr hilfreich, eine Buchung direkt mehreren Sollstellungen zuordnen zu können.

Wenn die offenen Beiträge aller markierten Sollstellungen exakt dem überwiesenen Betrag entspricht, könnte automatisch eine Splitbuchung erstellt werden, bei der die einzelnen Beiträge direkt den Sollstellungen zugeordnet sind.

Ich habe aber noch keine Idee, wie das sauber in der GUI umgesetzt werden könnte. Wenn man bei Mehrfach-Auswahl von Sollstellungen direkt in die Splittbuchung springt, finde ich das überraschend. Außerdem ist dann nicht klar, was mit den vorher eingegebenen Werten passiert.

Habt ihr Ideen dazu?

**Comment by lenilsas** [comment]
Habe ich in #547 mit umgesetzt
